.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+# See https://pre-commit.com for more information
```

Comment:
Maybe add isort, and you feel like it flake8?

Reply:
i already have `reorder_python_imports` in here, and black will enforce code style. don't see any changes required here